src/scrape_up/CovidInfo/covid_info.py

```diff
@@ -0,0 +1,88 @@
+import pandas as pd
+import requests
+from bs4 import BeautifulSoup
+
+"""
+        Class - `Covid Info`\n
+
+    | Methods                     | Details                                                                                              |
+    | --------------------------- | ---------------------------------------------------------------------------------------------------- |
+    | `.scrape()`                 | Returns the scraped data from the target website of all the countries in the form of a list          |
+    | `.findCountry()`            | Returns the data related to the specific country passed as a parameter into the function             |
+    | `.sortbycases()`            | Returns the list of datas of all the country, sorted according to the number of cases                |
+    | `.sortbydeaths()`           | Returns the list of datas of all the country, sorted according to the number of deaths               |
+    | `.totalcases()`             | Returns the total number of covid cases as of yet in the form of a string of numbers                 |
+    | `.totaldeaths()`            | Returns the total number of covid deaths as of yet in the form of a string of numbers                |
+"""
+
+class CovidInfo:
+  def __init__(self):
+        print(self)
+
+  def scrape(self):
+    url = 'https://www.worldometers.info/coronavirus/countries-where-coronavirus-has-spread/'
+    page = requests.get(url)
+    soup = BeautifulSoup(page.text, 'html.parser')
+    data = [['Country','Number of Cases', 'Deaths']]
+    data_iterator = iter(soup.find_all('td'))
+    while True:
+      try:
+        country = next(data_iterator).text
+        confirmed = str(next(data_iterator).text.replace(',','').replace(' ',''))
+        deaths = str(next(data_iterator).text.replace(',','').replace(' ',''))
+        continent = next(data_iterator).text
+
+        data.append([
+            country,
+            int(confirmed),
+            int(deaths),
+            continent
+        ])
+
+      except StopIteration:
+        break
+    return data
+
+
+  def findCountry(self, name):
+    data = self.scrape()
+    fc = [['Country','Number of Cases', 'Deaths']]
+    for i in data:
+      if i[0].lower() == name.lower():
+        fc.append([i[0],i[1],i[2]])
+
+        return fc
+    return 'Data not aviliable for value entered'
+
+  def sortbycases(self, rev):
+    data = self.scrape()
+    data.remove(['Country','Number of Cases', 'Deaths'])
+    data = sorted(data, key= lambda r:r[1],reverse=rev)
+    return data
```

Comment:
Implement try-exception in all the methods.

Reply:
Sir, I've implemented the changes as you prescribed. Please review the work. Thanks!